Assets/Scripts/Player/PlayerController.cs

```diff
@@ -9,6 +9,9 @@ public class PlayerController : MonoBehaviour {
     [SerializeField]
     private int currentHealth;
 
+    public int getMaxHealth() { return maxHealth; }

```

Comment:
I think these methods should be moved below the instance variables and have their names capitalized, but we don't have style guidelines at this point.